builder/xml_form_builder.module

```diff
@@ -580,14 +591,14 @@ function xml_form_builder_update_object(AbstractObject $object, array $associati
  * Applies xslt to incoming datastream.
  *
  * @param DOMDocument $source_document
- *   Document to be transformed
+ *   Document to be transformed.
  * @param xslt $transform
- *   Name of xslt to be applied
+ *   Name of xslt to be applied.
  *
  * @return DOMDocument
  *   Transformed document on success, original on failure.
  */
-function xml_form_builder_transform_metadata_datastream(DOMDocument $source_document, $transform = NULL) {
+function xml_form_builder_transform_metadata_datastream(DOMDocument $source_document, xslt $transform = NULL) {
```

Comment:
if `xslt` really a data type/class? Not sure about that. Where is that defined so? Seems to me that array_key_exists($transform, $transforms) denotes that $transform is just an array key value, so type hinting there should be wrong and also annotation `@params` for $transform?